doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -748,6 +758,27 @@ As a fallback mechanism for clock synchronization, the accessory must simultaneo
 The Fast Pair advertising frames make the affected Provider visible to nearby Seekers.
 Once one of the Seeker devices connects to the accessory and synchronizes the clock, the :c:member:`bt_fast_pair_fmdn_info_cb.clock_synced` callback is called to indicate that the Provider is no longer required to advertise the Fast Pair payload.
 
+.. _ug_bt_fast_pair_gatt_service_fmdn_info_callbacks_conn_authenticated:
+
+Connection authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The connected peer authentication is indicated by the :c:member:`bt_fast_pair_fmdn_info_cb.conn_authenticated` callback.
+This callback is triggered on a successful provisioning state read operation over the Beacon Actions characteristic.
+A successful operation proves that the connected peer has at least one of the Provider's Account Key.
+
+A typical use case for this callback is to authenticate the connected peer and allow them to read the firmware version to support firmware update intents on the Android platform.
+See :ref:`ug_bt_fast_pair_gatt_service_firmware_update_intent` for general information about the firmware update intent feature.
```

Comment:
The information was moved to `ug_bt_fast_pair_provisioning_register_firmware_update_intent`

Reply:
I have split it into two sentences to mention both links